Classes/Private/Manager/Objects/OCTMessageCall.m

```diff
@@ -32,13 +25,10 @@ - (NSString *)typeDescription
 {
     NSString *description;
     switch (self.callEvent) {
-        case OCTMessageCallEventDial:
-            description = @"Call dial";
-            break;
-        case OCTMessageCallEventEnd:
+        case OCTMessageCallEventAnswered:
             description = [[NSString alloc] initWithFormat:@"Call lasted %f seconds", self.callDuration];
             break;
-        case OCTMessageCallEventMissed:
+        case OCTMessageCallEventUnanswered:
             description = @"Call missed";
```

Comment:
Not "missed", but "unanswered". It can be outgoing call which cannot be missed to us.